frontend-ui/src/views/PipelineView.vue

```diff
@@ -167,7 +171,32 @@ async function loadData(limit: number, skip: number, filter?: string) {
 }
 
 async function loadLastRuns() {
-  // TODO: Load last runs
+  lastRunsLoaded.value = false
+  const schedule_names = Array.from(new Set(tasks.value.map((task) => task.schedule_name)))
+  loadingStore.startLoading('Loading last runs...')
+
+  const chunkSize = constants.TASKS_LOAD_SCHEDULES_CHUNK_SIZE
```

Comment:
I'm sorry but I don't get this logic purpose:
- why making batches of 5 requests? If the goal is to avoid overwhelming the server, then I would rather pause between every requests, maybe only 100ms or something like that
- why awaiting getDelay every time? To me it looks like we always wait even if we have only one chunck

Or do I miss something obvious?

Reply:
i was just trying to recreate the logic from here https://github.com/openzim/zimfarm/blob/0feac1c43a48c12079a7123f3fceb793f233d06f/dispatcher/frontend-ui/src/components/PipelineTable.vue#L189-L198


I didn't fully understand it but now that you mention, I could rewrite it this way instead.

Reply:
Indeed, looks like logic is already crap. Please fix this: make a small pause between each request for a schedule so that we do not overwhelm the backend. And if feasible (not sure how this is), update the UI as the results arrive (instead of updating them all at once when we have received all results).